lib/cisco_node_utils/interface.rb

```diff
@@ -1026,11 +1026,12 @@ def switchport_vtp_mode_capable?
     end
 
     def switchport_vtp
-      return nil unless switchport_vtp_mode_capable?
+      return default_switchport_vtp unless switchport_vtp_mode_capable?
```

Comment:
@glennmatthews We talked about doing that early on -- that if the interface class became too big or unwieldy that we should split them up. Maybe we're at that point now.